kallsyms/kallsyms.go

```diff
@@ -95,12 +95,10 @@ func NewSymbolizer() (*Symbolizer, error) {
 // an index suitable for storing in the `symbol` struct.
 func (m *Module) addName(name string) uint32 {
 	index := len(m.names)
-	l := len(name)
-	// Cap the length to 255 bytes so it fits a byte. Longest seen
-	// symbol so far is 83 bytes.
-	if l > 255 {
-		l = 255
-	}
+	l := min(
+		// Cap the length to 255 bytes so it fits a byte. Longest seen
```

Comment:
Nit: I'd move these comments above and keel this min call on single line.

Reply:
fixed with https://github.com/open-telemetry/opentelemetry-ebpf-profiler/pull/638/commits/7aab0a34150f5f51d7f8c9599ae182cac85fa524